dockerfiles/hack/run-integration.py

```diff
@@ -57,7 +57,9 @@
 
 # Messages to stdout
 STREAM_HANDLER = logging.StreamHandler(sys.stdout)
-STREAM_HANDLER.setFormatter(logging.Formatter(fmt=LOG_FMT, datefmt=LOG_DATEFMT))
+STREAM_HANDLER.setFormatter(
+    logging.Formatter(fmt=log_fmt(dry_run_option=DRY_RUN), datefmt=LOG_DATEFMT)
```

Comment:
So the `DRY_RUN` env var is not `true` or `false` ? interesting...

Reply:
TIL https://github.com/app-sre/qontract-reconcile/blob/master/dockerfiles/hack/run-integration.py#L189-L191